app/Filament/Resources/EmulatorResource.php

```diff
@@ -209,6 +214,11 @@ public static function table(Table $table): Table
                     ->default(false)
                     ->alignCenter(),
 
+                Tables\Columns\IconColumn::make('supports_toolkit')
+                    ->boolean()
+                    ->default(false)
+                    ->alignCenter(),
+
```

Comment:
Unrelated to this PR, but I can't edit the emulator records where I'd most likely want to set this because the download links aren't complete URLs.

![image](https://github.com/user-attachments/assets/c1015e27-3166-4796-9d86-98f699b44aa3)

Reply:
I think this current behavior is probably desirable. Absolute URLs are a bit safer for the view layer to handle, as they don't come with any assumptions about relative base paths.

Reply:
Maybe so, but we currently rely on that logic to determine whether to display an external or internal download link.
https://github.com/RetroAchievements/RAWeb/blob/f560b7bc2a04d17fbfad292243f9dc6170a2992c/resources/views/pages-legacy/download.blade.php#L49

![image](https://github.com/user-attachments/assets/6b62a277-9258-4730-b88b-dbd0a173e895)